internal/manifests/manifests_test.go

```diff
@@ -269,7 +269,7 @@ var _ = Describe("ClusterManifestTests", func() {
 			It("Does not accept a filename that does not contain a name before the extension", func() {
 				clusterID := registerCluster().ID
 				content := "{}"
-				fileName := ".yaml"
+				fileName := ".yaml.patch"
```

Comment:
shouldn't we test both cases? 
`.yaml` and `.yaml.something`

Reply:
The point is to check for the filename starting with period.
The regex already takes care of these cases.

We can possibly update the test to cover lots of cases but I'm not sure it's necessary in this case?

Reply:
/lgtm